arches_lingo/src/arches_lingo/App.vue

```diff
@@ -74,7 +74,7 @@ router.beforeEach(async (to, _from, next) => {
 
 <style scoped>
 main {
-    font-family: sans-serif;
+    font-family: system-ui, sans-serif;
```

Comment:
With this change, there's now a difference (on my local browser) between semibold and bold. Which means: we can preset the font weight for button labels to 600/semibold without it being too loud (it was looking the same as bold)

Then we'll have a default for button labels that looks good both in the controlled list manager (arches chrome) and here.

Reply:
Eh I'd say remove `system-ui`, as that varies from system to system and pulls us away from a unified experience. https://infinnie.github.io/blog/2017/systemui.html

Reply:
Yeah I saw that also, but I can't remember if I saw that before or after I made this addition, 😄 . I'll back it out. You have a good point we're building a unified experience, not a blog for a law office.